integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -125,16 +126,20 @@ def export(self, **kwargs):
         course_details = None
 
         LOGGER.info(
-            '[Integrated Channel] Beginning export of enrollments: {}'.format(list(enrollment_queryset.values()))
+            '[Integrated Channel] Beginning export of enrollments: {enrollments} for enterprise user: {user}'.format(
```

Comment:
being nitpicky but I was thinking we could standardize the way our logging looks a bit by having sth consistent looking like 

```
[Integrated Channel] [channel_name] [enterprise_customer_name_or_uuid]: <message>
```

of course it may depend on what you are logging. having a util function to log with such consistency may make it easier for us to search through and correlate log entries

if a piece of info is not available, you would simply drop it from the logging (to avoid failures)

feel free to only do enough as needed for this pr, just some thoughts on logging overall

Reply:
Agreed on this front. I think a logging util is definitely a good way to go to enforce consistency (though it doesn't work for methods we override, but we're manually adding the channel name to those anyways and we can refactor that later)

Reply:
yup esp the channel, since now we have multiple integrated channels, and sometimes it take a while to even ralize it's not even canvas!